.appveyor.yml

```diff
@@ -48,7 +47,5 @@ build_script:
   - ps: Push-AppveyorArtifact C:\Projects\flang\lib.zip
 
 test_script:
-  - cmd: copy lib\flangmain.lib %CONDA_INSTALL_LOCN%\Library\lib
-  - cmd: copy lib\flangrti.lib %CONDA_INSTALL_LOCN%\Library\lib
-  - cmd: copy lib\flang.lib %CONDA_INSTALL_LOCN%\Library\lib
-  - cmd: copy lib\ompstub.lib %CONDA_INSTALL_LOCN%\Library\lib
+  - cmd: cmake --build . --target install
+  - cmd: dir /s
```

Comment:
Please make sure to find a command that is not so verbose, or upload a file to artifacts so that the log is readable.

Reply:
This was just a test.